.evg.yml

```diff
@@ -167,6 +134,8 @@ tasks:
       - func: "run smoke test"
 
   - name: "publish-maven"
+    # TODO: remove this before merging eap branch back into main branch, to allow download center to update.
```

Comment:
Wondering if we should put a check in the publish-maven task that prevents it from functioning if eap.

Reply:
Yeah, sure I added some checks for publish-maven, publish-s3, download center, and ssdlc tasks.